### ISSUE TITLE
investigate the difficulty of Cmake-free build

### ISSUE THREAD

**Issue by chikashimiyama** [pd-externals/hoalight#14]
Is it possible to compile without CMake build?
Hoa light is header only and eigen is also header only. it means theoretically we just need a reference to the directly?

**Comment by chikashimiyama** [comment]
@danomatika 

Due to my limited knowledge of Makefile, it is not easy for me.

Currently, I'm making a static library with C wrapper using C++ 14 and CMake and link pd object project against that. 

I guess what I have to express in Makefile is to compile all my C++ code using -std=c++14 option and somehow link that to hoalight.c ... ?

as I wrote above, both hoalight and eigen are header only. to include those projects would be not so difficult.

**Comment by danomatika** [comment]
Yes, but I can do it later. It's more important that the object builds and works. Also, I'd rather have earplug working like zirkhrtf~ than this building without Cmake.

**Comment by danomatika** [comment]
Also, I'm on vacation for the next 2 weeks, so I won't be doing anything hands on with this for a while.

**Comment by chikashimiyama** [comment]
ok. hope I can solve all the important issues by then.
